AudioLink/Scripts/AudioLink.cs

```diff
@@ -1,8 +1,11 @@
 ﻿using UnityEngine;
-using VRC.SDKBase;
 using UnityEngine.UI;
 using System;
 
+#if VRC_SDK_VRCSDK3
```

Comment:
Nice, is this define documented somewhere though?

Reply:
https://docs.vrchat.com/docs/sdk-troubleshooting#im-getting-errors-related-to-sdk3-or-udon-in-a-project-using-sdk2-or-vrc_spatialaudiosource-in-a-project-using-sdk3

Reply:
ah, missed that blurb, thanks